modules/flowable-bpmn-converter/src/main/java/org/flowable/bpmn/converter/UserTaskXMLConverter.java

```diff
@@ -56,7 +57,9 @@
             new ExtensionAttribute(ATTRIBUTE_TASK_USER_CATEGORY),
             new ExtensionAttribute(ATTRIBUTE_FORM_FIELD_VALIDATION),
             new ExtensionAttribute(ATTRIBUTE_TASK_SERVICE_EXTENSIONID),
-            new ExtensionAttribute(ATTRIBUTE_TASK_USER_SKIP_EXPRESSION));
+            new ExtensionAttribute(ATTRIBUTE_TASK_USER_SKIP_EXPRESSION),
+            new ExtensionAttribute(ATTRIBUTE_TASK_USER_SKIP_EXPRESSION),
```

Comment:
Curious, why is `ATTRIBUTE_TASK_USER_SKIP_EXPRESSION` defined twice?  Lines 60 and 61.

Reply:
Good catch, was a copy/paste error, I removed the second one.